app/decorators/user_ticket_decorator.rb

```diff
@@ -0,0 +1,11 @@
+class UserTicketDecorator < Draper::Decorator
+  delegate_all
+
+  def status
```

Comment:
This method already exists on the master with another purpose. You can rename or find a way to reuse.